plotter/plot_classes/roc_plot.py

```diff
@@ -28,12 +29,15 @@ def plot(self):
                 ds = hdf_file[sample_config.df_name]
 
                 target_label = self.config.target_label
-                tagger_output = self.config.tagger_output
+
+                # get attribute name for GNN ej score
+                keys_list = list(ds.dtype.fields.keys())
+                pDisp = keys_list[-2]
```

Comment:
not sure about this logic here, it could happen in the future that the ordering of the columns in the h5 files changes. it would be better to have something in the yaml that sets the name to 'GN2ej_pdispjet' or something else

Reply:
I did that so it is more generalizable. Originally, you specified in the dataframe the name "'GN2ej_pdispjet", but if one changes the name of the yaml file, the "GN2ej" part changes. So, doing this allows it to always get the correct name for the signal discriminant, no matter what the file naming is.